.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
+</Workspace>
```

Comment:
do you know why this was added?

Reply:
hmm i'm not sure, i think it was automatically added when i created the changelog markdown file - but it doesn't seem relevant. maybe i can delete it?

Reply:
I'm in favor of removing if possible! 👍